plugins/modules/mso_schema_template_anp_epg_annotation.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Samita Bhattacharjee (@samiib) <samita@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: mso_schema_template_anp_epg_annotation
+short_description: Manage EPG Annotations on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Endpoint Group (EPG) Annotations on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.0 (NDO v4.2) and later.
+author:
+- Samita Bhattacharjee (@samiib)
+options:
+  schema:
+    description:
+    - The name of the schema.
+    type: str
+    required: true
+  template:
+    description:
+    - The name of the template.
+    type: str
+    required: true
+  anp:
+    description:
+    - The name of the Application Network Profile (ANP).
+    type: str
+    required: true
+  epg:
+    description:
+    - The name of the EPG to manage.
+    type: str
+    required: true
+  annotation_key:
+    description:
+    - The key of the Annotation object.
+    type: str
+    aliases: [ key ]
+  annotation_value:
+    description:
+    - The value of the Annotation object.
+    type: str
+    aliases: [ value ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment: cisco.mso.modules
+notes:
+- The O(schema) and O(template) must exist before using this module in your playbook.
+  Use M(cisco.mso.mso_schema_template) to create the schema template.
+- The O(anp) must exist before using this module in your playbook.
+  Use M(cisco.mso.mso_schema_template_anp) to create the ANP.
+- The O(epg) must exist before using this module in your playbook.
```

Comment:
Shall we actually combine all of them?
- The O(schema), O(template), O(anp) and O(epg) must exist before using this module in your playbook.

Reply:
Done 👍